content/blog/2023/11/2023-11-10-Jenkins-Contributor-Summit-in-Brussels.adoc

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: "Jenkins Contributor Summit in Brussels"
+tags:
+- events
+- community
+- contribute
+author: jmMeessen
+opengraph:
+  image: /images/hacktoberfest/hacktoberfest_2023_social_600x300.png
+---
+
+
+We are happy to announce that we will hold a **Jenkins Contributor Summit** in Brussels prior to FOSDEM. 
+
+link:https://fosdem.org/2024/[FOSDEM] is a free event for software developers to meet, share ideas and collaborate. 
+Every year, thousands of developers of free and open source software from all over the world gather at the event in Brussels. 
+It takes place on Saturday 3 & Sunday 4 February 2024.
+
+== What is a Jenkins Contributor Summit?**
+The Jenkins Contributor Summit brings together current and future contributors to the Jenkins project. 
+At this event  we will talk about the current state of the project and its future evolution. 
+It is a great opportunity to meet "in real life" fellow actors of the Jenkins community.
+
+It is a day's event (10h00 to 17h00) with presentations on the "State of Jenkins", Project/SIG updates, and various key projects as well as "Ignite Talks" or demos.
```

Comment:
```suggestion
It is a day-long event (from 10:00 AM to 5:00 PM) featuring presentations on the "State of Jenkins", Project/SIG updates, discussions on various key projects, "Ignite Talks", and demos.
```

Reply:
Thank you. I agree with most suggestion but not the time format. The event takes place in Europe (even England uses 24h time format) and FOSDEM time related announcements are in 24h format.